rita_bin/src/client.rs

```diff
@@ -60,6 +60,9 @@ fn main() {
     })
     .expect("Error setting Ctrl-C handler");
 
+    //Restart babeld
+    KI.setup_babel();
```

Comment:
this needs a doc comment

/// Because Rita clears and sets up new Wireguard Tunnels on every restart Babel, which was attached and listening to /// the old tunnels is now in an incorrect state. We must either restart babel or empty it's interfaces list so that the newly /// created wireguard tunnels can be re-added by this instance of Rita. Due to errors in babel (see git history there) 
/// restarting is the way to go as removing dead interfaces often does not work

Reply:
fixed